net/netutil/udp_gro_forwarding_linux.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package netutil
```

Comment:
This PR is the classic example of why "util" grabbag packages turn bad: now derper and the CLI depend on the ethtool package for no reason, because they depended on some other utility in "netutil".

I think we should probably nuke the ethtool package entirely (or require with a new deptest it be only network stuff that has no deps outside of tailscale.com/std).

We can move this to some new package with just this one function. You'll have to think of a package name (sometimes I'm jealous of Unison: https://www.unison-lang.org/learn/the-big-idea/) but it can be ugly. Like `net/netutil/netkernelconf` or something.

Reply:
(to be clear, it's not your job to nuke all of netutil in this PR... we should do that later over time. Let's just not make it worse.)

Reply:
Updated in https://github.com/tailscale/tailscale/pull/10071/commits/e32bb4609c30019e645c9f76152a835c8504fbcb